content/blog/pkgdown-2-0-0/index.Rmd

```diff
@@ -0,0 +1,190 @@
+-->
+```
+We're very chuffed to announce the release of [pkgdown](https://pkgdown.r-lib.org) 2.0.0.
+pkgdown is designed to make it quick and easy to build a website for your package.
+Install it with:
+
+```{r, eval = FALSE}
+install.packages("pkgdown")
+```
+
+This is a massive release that brings a major visual refresh and a huge number of improvements.
+This release would not have been possible without pkgdown's newest author [Maëlle Salmon](https://masalmon.eu), who was the powerhouse behind many of the improvements in this release, particularly the switch to bootstrap 5, improved customisation, and implementation of local search.
+
+There are way too many changes to describe individually here, so this post will focus on the most important new features:
+
+-   The new template that uses bootstrap 5.
+-   The most important new ways to customise your site.
+-   Some of the biggest changes to code display.
+-   A grab bag of other cool features.
+
+See the [release notes](https://github.com/r-lib/pkgdown/blob/main/NEWS.md) for a complete list of everything that's changed.
+
+```{r setup}
+library(pkgdown)
+```
+
+## Bootstrap 5
+
+pkgdown comes with a refreshed template that uses [Bootstrap 5](https://getbootstrap.com/docs/5.1/getting-started/introduction/), an major update over the previous [Bootstrap 3](https://getbootstrap.com/docs/3.4/).
+Because this is a major change, you'll need to opt-in by setting the `boostrap` version in your `_pkgdown.yml`:
+
+``` yaml
+template:
+  bootstrap: 5
+```
+
+(The old bootstrap 3 template is superseded; it will continue to work for some time, but it won't gain any new features and we encourage you to switch to the new template the next time you're working on your package.)
+
+The new theme includes:
+
+-   A number of minor improvements to accessibility, including a larger font size, greater use of `aria` attributes, and [accessible syntax highlighting colour scheme](https://apreshill.github.io/rmda11y/arrow.html), designed by Alison Hill.
+
+-   Support for new features like in-line footnotes and [tabsets in articles](https://bookdown.org/yihui/rmarkdown-cookbook/html-tabs.html).
+
+-   Since so many packages have lovely logos, the package logo is now displayed on every page and made even more prominent on the home page.
+
+As an added incentive to upgrade your template to Bootstrap 5, you'll get site search for "free": pkgdown now supports searching with no external dependencies and no set-up.
+Learn more in `vignette("search")`.
+
+## Customisation
+
+The new template is also much easier to customise.
+A few of the most important features are noted below to whet your appetite; learn more in `vignette("customise")`.
+
+-   You can now easily change the overall visual appearance by picking a bootswatch theme:
+
+    ``` yaml
+    template:
+      bootstrap: 5
+      bootswatch: cyborg
+    ```
+
+    Or by selectively overriding the "bslib" variables used to generate the CSS:
```

Comment:
should bslib be introduced somewhere earlier in this section or the one before that?